syscallf/zsyscall_linux_386.go

```diff
@@ -0,0 +1,17 @@
+// mksyscall.pl -l32 -tags linux,386 syscall_linux.go syscall_linux_386.go
+// Code generated by the command above; DO NOT EDIT.
+
+//go:build linux && 386
+
+package syscallf
+
+import "syscall"
+
+func InotifyRmWatch(fd int, watchdesc int) (success int, err error) {
+	r0, _, e1 := syscall.RawSyscall(syscall.SYS_INOTIFY_RM_WATCH, uintptr(fd), uintptr(watchdesc), 0)
+	success = int(r0)
+	if e1 != 0 {
+		err = errnoErr(e1)
```

Comment:
I think this conversion is not needed, we could actually return e1, since it's syscall.Errno and implements error interface, correct me if I am wrong

Reply:
Yes, it implements error interface.